src/App.tsx

```diff
@@ -1,8 +1,59 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 
-const App = () => (
-  <h1>Dynamic list of posts</h1>
+import * as api from './api';
+
+const App = () => {
+  const [posts, setPosts] = useState([]);
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const loadData = async() => {
+    setIsLoading(true);
+
+    const postsFromServer = await api.getPosts();
```

Comment:
Would be great to have type here

Reply:
@YuriiHoliuk I typed the state and the API method
Should I still add type to the variable?